packages/react/src/components/icon/icon.stories.tsx

```diff
@@ -1,35 +1,30 @@
-import { Color } from '@onfido/castor';
 import { iconNames } from '@onfido/castor-icons';
 import { Icon, IconProps } from '@onfido/castor-react';
 import React from 'react';
-import { Meta, omit, Story, storyOf, tokens } from '../../../../../docs';
-
-const colors = Object.keys(tokens).reduce((accumulator, name) => {
-  const [, color] = name.match(/^--ods-color-([a-z0-9-]+)$/) ?? [];
-  return color ? [...accumulator, color] : accumulator;
-}, []) as Color[];
+import { colors, Meta, omit, reactMatrix, Story } from '../../../../../docs';
 
 const [firstIconName] = iconNames;
 
 export default {
   title: 'React/Icon',
   component: Icon,
   argTypes: {
-    ...omit<IconProps>('className', 'style'),
-    name: { control: { type: 'select', options: iconNames } },
     color: { control: { type: 'select', options: colors } },
   },
   args: {
     name: firstIconName,
   },
 } as Meta<IconProps>;
 
-export const Playground: Story<IconProps> = (props: IconProps) => (
-  <Icon {...props} />
-);
+export const Playground: Story<IconProps> = (props) => <Icon {...props} />;
 
-export const Name = storyOf(Icon, 'name', iconNames, {
-  display: 'grid',
-  labelMode: 'after',
-});
+export const Name = reactMatrix(Icon, { name: iconNames }, (props) => (
```

Comment:
I have noticed that on core stories of a list of all icons, a code example shows icon code + icon name:

<img width="351" alt="Screenshot 2021-02-12 at 10 39 37" src="https://user-images.githubusercontent.com/20243687/107758322-9ddec180-6d1e-11eb-9105-588b66d92dff.png">

But for React it only shows icon code:

<img width="312" alt="Screenshot 2021-02-12 at 10 38 05" src="https://user-images.githubusercontent.com/20243687/107758250-86073d80-6d1e-11eb-81f7-ecff55dc802f.png">

Is this what you were expecting?

Reply:
Yea I don't transform React source code.
I could if you think that's valuable, we could remove `key` too for instance.

Reply:
I actually think both "key" here and also text on core stories are not needed (especially those - why these would be handy?) 🤔

Also, these to me are minors. If too difficult to remove - then it is fine.

Reply:
I don't think it's too difficult, it would just mean running prettier on React code too.
I'll see what I can do.

Reply:
Removing `key` from React source now, and fixed the bug alongside the `children` for Button below.
Now it consistently shows the actual source, however it does look weird:
![image](https://user-images.githubusercontent.com/18623773/109502611-1cfb2600-7a91-11eb-9acc-8efb51a6845d.png)

I could support per-story source transform, but I guess it's ok to have actual source code show up?
Less magic?

Reply:
Probably yes, just that `{' '}` looks odd.

Reply:
Now also replacing `{' '}` instances.
The line break should do the same thing.